File: MetricsPython/app/main/application/dtos/export_aggregated_logs_csv_dto.py
class ExportAggregatedLogsCsvDto:
    def __init__(self, file_name: str, aggregated_log_date_type: str, device_id: int, field_id: int):
        self.file_name = file_name
        self.aggregated_log_date_type = aggregated_log_date_type
        self.device_id = device_id
        self.field_id = field_id

File: DeviceMetricsOrchestrator/Application/DTOs/Metrics/ExportAggregatedLogsCsvDTO.cs
using Domain.Enums;
using System.Text.Json.Serialization;

namespace Application.DTOs.Metrics
{
    public class ExportAggregatedLogsCsvDTO
    {
        public string? FileName { get; set; }

        [JsonConverter(typeof(JsonStringEnumConverter))]
        public AggregatedLogDateType AggregatedLogDateType { get; set; } 
        public int DeviceId { get; set; }
        public int FieldId { get; set; }
    }
}
